src/components/Header/Header.css

```diff
@@ -0,0 +1,13 @@
+.header {
+  text-align: center;
+}
+
+.header__welcome {
+  margin: 0;
```

Comment:
Do we have to have a global reset with margin and padding 0?

Reply:
Do you mean like css reset or normalize.css? We could do that, though I think they're kind of falling out of vogue with the idea that it's much less overhead to not have redundant rules, rules for elements you aren't even using etc. and the resets all come with their own different defaults to learn anyway. Since this is a fairly small project I think I favour just being explicit about margins, padding and font sizes in the BEM classes, because then it takes the guess work out for another developer looking at the styles.